src/SFA.DAS.Roatp.CourseManagement.Jobs/Functions/ReloadStandardsCacheFunction.cs

```diff
@@ -0,0 +1,41 @@
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Roatp.CourseManagement.Jobs.Infrastructure.ApiClients.RoatpV2Api;
+using SFA.DAS.Roatp.CourseManagement.Jobs.Infrastructure.ApiClients.RoatpV2Api.Models;
+using SFA.DAS.Roatp.CourseManagement.Jobs.Infrastructure.ApiClients.StandardsApi;
+
+namespace SFA.DAS.Roatp.CourseManagement.Jobs.Functions
+{
+    public  class ReloadStandardsCacheFunction
+    {
+        private readonly IGetActiveStandardsApiClient _getActiveStandardsApiClient;
+        private readonly IReloadStandardsApiClient _reloadStandardsApiClient;
+
+
+        public ReloadStandardsCacheFunction(IGetActiveStandardsApiClient getActiveStandardsApiClient, IReloadStandardsApiClient reloadStandardsApiClient)
+        {
+            _getActiveStandardsApiClient = getActiveStandardsApiClient;
+            _reloadStandardsApiClient = reloadStandardsApiClient;
+        }
+
+
+        [FunctionName(nameof(ReloadStandardsCacheFunction))]
+        public  async Task Run([TimerTrigger("%ReloadStandardsCacheSchedule%")] TimerInfo myTimer, ILogger log)
+        {
+
+            log.LogInformation($"ReloadStandardsCacheFunction function started");
+
+            var standardList = await _getActiveStandardsApiClient.GetActiveStandards();
```

Comment:
On non-success response the base class returns default which means standardList can be null.

Reply:
now deals with null response explicitly